iceberg/src/main/scala/com/nvidia/spark/rapids/iceberg/spark/GpuSparkReadOptions.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.nvidia.spark.rapids.iceberg.spark
+
+object GpuSparkReadOptions {
+  // Controls whether to allow reading timestamps without zone info
+  val HANDLE_TIMESTAMP_WITHOUT_TIMEZONE = "handle-timestamp-without-timezone"
```

Comment:
why can we not use https://github.com/apache/iceberg/blob/84741867b9220f47b8e9a83edfb1be14d40cbefc/spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkWriteOptions.java#L58 for this constant?